### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jnm2**
Ready for review. Waiting for the other outstanding PRs so that a more up-to-date version can be specified.

@sharwell The minimal example is what I'm going to be copying in pasting from all the time, if this is acceptable to you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,22 +36,35 @@ IL weaver for adding nullability annotations to .NET Framework and .NET Standard
 
 ```xml
 <PropertyGroup>
-  <!-- Specifies the version of this rewriter to use -->
-  <TunnelVisionLabsReferenceAssemblyAnnotatorVersion>1.0.0-alpha.50</TunnelVisionLabsReferenceAssemblyAnnotatorVersion>
-  <!-- Specifies the version of Microsoft.NETCore.App.Ref to obtain nullability information from -->
-  <AnnotatedReferenceAssemblyVersion>3.0.0-preview9-19423-09</AnnotatedReferenceAssemblyVersion>
+  <!-- Specifies the version of this rewriter to use. -->
+
+  <TunnelVisionLabsReferenceAssemblyAnnotatorVersion>1.0.0-alpha.69</TunnelVisionLabsReferenceAssemblyAnnotatorVersion>
```
Comment (sharwell) [inline README.md]: I think the blank line was meant to go after each property?

Comment (jnm2) [reply README.md]: Absolutely, thanks! I have no idea what happened.

**Comment by jnm2** [comment]
@sharwell CI builds don't go to nuget.org. Not sure what I was thinking this should wait for.
 Do you want to do a new nuget.org release before this PR merges so that I can know which version to put in the readme, or should I leave it at 69 which didn't have all the outstanding issues closed?

**Comment by sharwell** [comment]
I published 1.0.0-alpha.77 for you. It's like 2 clicks on AppVeyor to publish a master branch commit to NuGet thanks to @AArnott's Git versioning tools.

**Pull request merged**